python/lsst/ts/standardscripts/auxtel/daytime_checkout/pnuematics_checkout.py

```diff
@@ -0,0 +1,159 @@
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        # This script does not require any configuration
+        pass
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = 5.0 * 60  # Approximate 5min to completion
+
+    async def run(self):
+
+        await self.checkpoint("Opening pnuematics valves")
+
+        await self.atcs.open_valves()
+        pressure = await self.atcs.rem.atpneumatics.tel_mainAirSourcePressure.next(
+            flush=True, timeout=STD_TIMEOUT
+        )
+        if pressure.pressure > 300000:
+            self.log.info(
+                f"Air pressure is {pressure.pressure:0.0f} Pascals, which is fine."
+            )
+        else:
+            raise AssertionError(
+                f"Air pressure is {pressure.pressure}, which is too low. It needs "
+                f"to be between ~275790 and ~413000 Pascals (40 and 60 PSI). \n "
+                f"Check that compressor and dryer are running. Then check that the "
+                f"regulator inside the pier is set correctly. "
+            )
+
+        # Turn on ATAOS correction(s).
+
+        await self.checkpoint("Turning on ATAOS coerrections")
+
+        cmd = await self.atcs.rem.ataos.cmd_enableCorrection.set_start(
+            m1=True, hexapod=True, atspectrograph=True
+        )
+        cmd_time = get_topic_time_utc(cmd)
+        self.log.info(f"ATAOS corrections enabled -- {cmd.result} at {cmd_time} UT")
+
+        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(timeout=5)
+        self.log.info(
+            f"M1 Air pressure with enabled ATAOS corrections is {m1_pressure.pressure:0.0f} Pascals"
+        )
+
+        await self.checkpoint("Turning off ATAOS coerrections")
+
+        # Turn off ATAOS correction(s)
+        cmd = await self.atcs.rem.ataos.cmd_disableCorrection.set_start(
+            m1=True, hexapod=True, atspectrograph=True
+        )
+        cmd_time = get_topic_time_utc(cmd)
+        self.log.info(f"Corrections disabled -- {cmd.result} at {cmd_time} UT")
+
+        await self.checkpoint("Lowering Mirror to hardpoints")
+
+        # Lower mirror back on hard points
+        cmd = await self.atcs.rem.atpneumatics.cmd_m1SetPressure.set_start(pressure=0)
+        m1_pressure = await self.atcs.rem.atpneumatics.tel_m1AirPressure.aget(
+            timeout=STD_TIMEOUT
+        )
+        self.log.info(
+            f"M1 air pressure when lowered back on hardpoints is {m1_pressure.pressure:0.0f} Pascals"
+        )
+
+        await self.checkpoint("Exercising Mirror Covers and Vents")
+
+        # Open mirror covers and vents
+        await self.atcs.open_m1_cover()
+        await self.atcs.open_m1_vent()
+
+        # Close mirror covers and vents
+        await self.atcs.close_m1_cover()
+        await self.atcs.close_m1_vent()
+
+    async def assert_feasibility(self):
```

Comment:
Are you using this anywhere?

Reply:
Method call at top of run added to all daytime_checkout scripts.